alf/utils/metric_utils.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+import tensorflow_probability as tfp
+from tf_agents.policies.tf_policy import Base
+from tf_agents.eval.metric_utils import eager_compute as tfa_eager_compute
+
+
+class Policy(Base):
+    """Wrap a action fn to policy """
+
+    def __init__(self, time_step_spec,
+                 action_spec, policy_state_spec, action_fn):
+        super(Policy, self).__init__(
+            time_step_spec=time_step_spec,
+            action_spec=action_spec,
+            policy_state_spec=policy_state_spec)
+        self._action_fn1 = action_fn
```

Comment:
why '_action_fn1' instead of '_action_fn'?

Reply:
already exist attribute `_action_fn` in parent class